pkg/component/etcd/testdata/monitoring_alertingrules_important_multinode_with_backup.yaml

```diff
@@ -102,29 +102,25 @@ tests:
     - exp_labels:
         pod: etcd
         role: test
-        job: kube-etcd3-backup-restore-main
-        kind: Incr
```

Comment:
... and that leads to losing some labels that the alert used to have before.

Is this intentional? I think keeping those labels (to differentiate later when aggregating the alerts by kind) is useful.